website/screens/components/nav-tabs/specs/NavTabsSpecsPage.tsx

```diff
@@ -0,0 +1,279 @@
+import {
+  DxcBulletedList,
+  DxcFlex,
+  DxcTable,
+  DxcParagraph,
+} from "@dxc-technology/halstack-react";
+import DocFooter from "@/common/DocFooter";
+import Figure from "@/common/Figure";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import Image from "@/common/Image";
+import Code from "@/common/Code";
+import navTabsSpecs from "./images/navTabs_specs.png";
+import navTabsAnatomy from "./images/navTabs_anatomy.png";
+import navTabsStates from "./images/navTabs_states.png";
+
+const sections = [
+  {
+    title: "Specifications",
+    content: (
+      <Figure caption="Nav tabs design specifications">
+        <Image src={navTabsSpecs} alt="Nav tabs design specifications" />
+      </Figure>
+    ),
+  },
+  {
+    title: "States",
+    content: (
+      <>
+        <DxcParagraph>
+          Tabs can get different states based on user interaction. States:{" "}
+          <strong>enabled</strong>, <strong>hover</strong>,{" "}
+          <strong>focus</strong>, <strong>active</strong> and{" "}
+          <strong>disabled</strong>.
+        </DxcParagraph>
+        <Image src={navTabsStates} alt="image" />
+      </>
+    ),
+  },
+  {
+    title: "Anatomy",
+    content: (
+      <>
+        <Image src={navTabsAnatomy} alt="image" />
+        <DxcBulletedList type="number">
+          <DxcBulletedList.Item>Container</DxcBulletedList.Item>
+          <DxcBulletedList.Item>Default text label</DxcBulletedList.Item>
```

Comment:
Why don't we call it just "Label" as in the other components?

![image](https://user-images.githubusercontent.com/44321109/224764604-280eb3db-490a-4a1a-a529-e3045b072aab.png)

![image](https://user-images.githubusercontent.com/44321109/224764866-b09b155f-105e-4688-8683-179a723da5dd.png)

Reply:
I think it does not refer to the tab label, but to the tab container. It is the same case as in tabs specs:

![image](https://user-images.githubusercontent.com/10975289/224774042-72f1e759-54c5-442f-bea5-805781305762.png)